packages/tophide/tophide.1.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "andy.ray@ujamjar.com"
+authors: "Martin Jambon"
+homepage: "https://github.com/mjambon/tophide"
+bug-reports: "https://github.com/mjambon/tophide/issues"
+dev-repo: "git+https://github.com/mjambon/tophide"
+build: make
```

Comment:
```suggestion
license: "BSD-3-Clause"
build: make
```
I think this is the one in the repo

Reply:
this is not the same to the letter (close 3 is missing mention of the contributors) but close enough, i think the spirit is the same.